conductor-config-ci.yaml

```diff
@@ -1,3 +1,6 @@
+dpki:
+  device_seed_lair_tag: ci
+  no_dpki: true
```

Comment:
Separately, this should also go into the "non CI" version of this file that gets used with Holoports. Currently, only the configured network services should be different I think

Reply:
just removed this from the ci version for now